### PR TITLE
clickable mail id done

### PR DESCRIPTION
**Pull request by rrkas**

### CODE REVIEW

File: lib/pages/AboutUsPage.dart/AboutUs.dart [hunk 1]
```diff
@@ -31,10 +33,25 @@ class _AboutUsState extends State<AboutUs> {
...
+                    text: "Thank you for using KIDE, the KIIT guide by the students, for the students! " +
+                        "We're a small team of Flutter Developers who decided to take the initiative to create an app which could serve as a hub for all the day-to-day student " +
+                        "tasks as well as general tasks related to our esteemed university. We hope that our work reflects " +
+                        "this ideology and you get to enjoy a seamless user experience with minimal issues. However, " +
+                        "we'd love to hear from you. Feel free write your suggestions and queries to us at - ",
+                  ),
+                  TextSpan(
+                    text: 'kide.kiit@gmail.com',
+                    style: TextStyle(decoration: TextDecoration.underline, color: Colors.blue),
+                    recognizer: TapGestureRecognizer()..onTap = () => launch('mailto:kide.kiit@gmail.com'),
```
Comment (emem365) [inline lib/pages/AboutUsPage.dart/AboutUs.dart]: A better practice would be to check `canLaunch()` before the launch. Rest of the work seems good 👍

Comment (rrkas) [reply lib/pages/AboutUsPage.dart/AboutUs.dart]: Should i resend a pull request with improvements...

Comment (emem365) [reply lib/pages/AboutUsPage.dart/AboutUs.dart]: No need for a new PR. You can just commit your changes to this branch and they would reflect here.

Comment (Cryptonex7) [reply lib/pages/AboutUsPage.dart/AboutUs.dart]: > A better practice would be to check `canLaunch()` before the launch. Rest of the work seems good 👍

Good catch @emem365, @rrkas we will be awaiting these changes.

Comment (rrkas) [reply lib/pages/AboutUsPage.dart/AboutUs.dart]: Sorry sir. I am a beginner. I didn't know that. I'm on it.

Comment (Cryptonex7) [reply lib/pages/AboutUsPage.dart/AboutUs.dart]: Sue @rrkas, dont worry - this is the best learning opportunity for beginners like you. You will certainly be benefitted through this.

Comment (rrkas) [reply lib/pages/AboutUsPage.dart/AboutUs.dart]: committed and pushed. please check.

**Comment by Cryptonex7** [comment]
@A4k9 Can you please review this PR?

**Review by sambitraze — APPROVED** [review]

**Review by vanshBhatia-A4k9 — APPROVED** [review]

**Review by Cryptonex7 — APPROVED** [review]
Works as reported by @sambitraze!

Ready for merge. Marking as hacktoberfest-accepted

Congratulations @rrkas, you will receive your certificate soon!

**Comment by rrkas** [comment]
Thank you sir. I completed all 4 PRs.
How soon will I receive my certificate and goodies...

**Comment by Cryptonex7** [comment]
## Congratulations @rrkas,
We are honoured to have you contributing to Kide.

We would like you to do more such contributions!

Please accept this token of acknowledgement from us and feel free to upload this to your LinkedIn profile or other portfolios.

Rohnak Agarwal

**Comment by rrkas** [comment]
Thanks a lot sir

**Pull request merged**